plasma_framework/truffle-config.js

```diff
@@ -14,7 +14,7 @@ module.exports = {
             host: process.env.ETH_CLIENT_HOST || '127.0.0.1',
             port: process.env.ETH_CLIENT_PORT || 8545,
             from: process.env.DEPLOYER_ADDRESS,
-            gas: 6000000,
+            gas: 10000000,
```

Comment:
why do we need to increase the gas?

Reply:
I did get out of gas error when I tried to deploy locally.

Reply:
ah...okay, just check current deployment gas and indeed is slightly exceeding 6m. Can you just remove that line instead. It would be using default truffle gas cost of ~6.7m which is ganache default. I would not want to increase to 10m as that is too high and I don't think any chain is using such high gas limit at the moment.